api/src/pcapi/core/external_bookings/api.py

```diff
@@ -91,7 +91,10 @@ def book_event_ticket(
         provider.bookingExternalUrl, json=payload.json(), headers={"Content-Type": "application/json"}
     )
     _check_external_booking_response_is_ok(response)
-    parsed_response = pydantic.parse_obj_as(serialize.ExternalEventBookingResponse, response.json())
+    try:
+        parsed_response = pydantic.parse_obj_as(serialize.ExternalEventBookingResponse, response.json())
+    except pydantic.ValidationError as err:
+        raise exceptions.ExternalBookingException(f"External booking failed. Could not parse response: {err}")
```

Comment:
est-ce qu'on appelerait pas la route pour annuler les billets ici ?

Reply:
pas la peine, si on arrive ici c'est que la billetterie distante nous a renvoyé un 200.

par contre, ce serait bien de changer le message d'erreur pour faire comprendre que tout va bien, comme `f"Could not parse external booking cancel response: {exc}"`

Reply:
On veut bien renvoyer une erreur si la validation ne réussit pas non ?
pour moi on considère que le front ne supporte pas l''envoie de code barre de plus de 100 caractères